packages/graphql-fixtures/package.json

```diff
@@ -26,7 +26,7 @@
   "dependencies": {
     "@types/faker": "^4.1.2",
     "faker": "^4.1.0",
-    "graphql-tool-utilities": "^0.7.4"
+    "graphql-tool-utilities": "0.7.4"
```

Comment:
You are not supposed to change this, lerna should be managing these dependencies

Reply:
how does this magic dependency management work?

Reply:
ok, restored the original version specs (and learned how to use lerna!)